.travis.yml

```diff
@@ -22,13 +22,15 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+dist: trusty
+
 addons:
-  postgresql: 9.4
+  postgresql: 9.6
```

Comment:
I did long time ago, but I think was related to `JSONB` and the queries I can do with it.
I have to check again to be sure if it's really needed.